### PR TITLE
 add automation for clusterlogforwarder

### PR DESCRIPTION
**Pull request by QiaolingTang**
Add automation scenarios for clusterlogforwarder. @anpingli PTAL, thanks.
cases: https://url.corp.redhat.com/3d74e5e
Runner-v3 logs: https://url.corp.redhat.com/31636ba

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
LGTM

File: testdata/logging/clusterlogforwarder/fluentd/secure/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    metadata:
+      labels:
+        logging-infra: "fluentdserver"
+        provider: aosqe
+        component: "fluentdserver"
+    spec:
+      containers:
+      - name: "fluentdserver"
+        image: "quay.io/openshifttest/fluentd@sha256:7eece00d1bc784ac1e9722b2580911cd3ead5afd740dad6594be945b3b1dd884"
+        imagePullPolicy: "Always"
```
Comment (liangxia) [inline testdata/logging/clusterlogforwarder/fluentd/secure/deployment.yaml:29]: Do we need to always pull the image ?

Comment (QiaolingTang) [reply testdata/logging/clusterlogforwarder/fluentd/secure/deployment.yaml:29]: No, before using the image digest, I used the `latest` tag and I wanted the image is always the latest, so I set `Allways` here, I can update it to "IfNotPresent".

Comment (QiaolingTang) [reply testdata/logging/clusterlogforwarder/fluentd/secure/deployment.yaml]: @liangxia updated, PTAL.

**Pull request merged**